pkg/util/utils_test.go

```diff
@@ -227,4 +230,11 @@ func TestNonWritableDirectory(t *testing.T) {
 
 	os.Chmod(testDir, 0400) // make dir read-only
 	assert.False(t, IsDirectoryWritable(testDir))
+
+}
+
+func isAppVeyorUser() bool {
+	// Checking for $env:APPVEYOR does not work !
```

Comment:
@gbraad Even setting it `APPVEYOR: True` to config file not working? if not then we should remove that addition.

Reply:
I had removed this in a later push... seems it is not at github, but onlu on gitlab. (must have timed out on push)... repushing